tcms/kiwi_auth/admin.py

```diff
@@ -144,6 +144,19 @@ def delete_model(self, request, obj):
         delete_user(obj)
 
 
+class GroupAdmin(admin.ModelAdmin):
```

Comment:
Group edit usually contains 2 widgets for permissions all & selected which are displayed side by side. Here there's only 1 so something is a bit messed up.

Reply:
Still working on this